plugins/node/instrumentation-kafkajs/src/instrumentation.ts

```diff
@@ -71,7 +71,7 @@ export class KafkaJsInstrumentation extends InstrumentationBase<KafkaJsInstrumen
 
     const module = new InstrumentationNodeModuleDefinition(
       'kafkajs',
-      ['>=0.1.0 <3'],
+      ['>=0.3.0 <3'],
```

Comment:
please also update the supported version range in the [README](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/instrumentation-kafkajs#supported-versions)